com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -544,5 +544,16 @@ public Vector2Int GetLastScaledSize()
         {
             return m_LastScaledSize;
         }
+
+        /// <summary>
+        /// Returns true if the current configuration should allow for low res transparency, returns false otherwise.
+        /// </summary>
+        /// <returns>True if the current configuration should allow for low res transparency, returns false otherwise.</returns>
+        public bool AllowLowResTransparency()
```

Comment:
This is actually problematic as it will pop when going below a certain threshold (with actually dynamic res and not hardcoded dimension).  Say we over around the minimum threshold, it will constantly go between full res and low res, which will be a big jump. 

We should instead probably just clamp the min resolution the low res transparent can reach. So we don't pop, but stop scaling down. 

Say the last accepted low res resolution is 360p (i.e. main res 720p), if the resolution with dynamic res drops below 720 we still use whatever is lower between 360p and the main res. This will still pop at very very low resolutions (less than half the percentage that is allowing the low res), but it should be way less often and jarring (since at that point we are likely an hodgepodge of pixels anyway and therefore the settings are likely to be set so that it won't happen in a shipped product)

Reply:
This is doable, but obviously more work :) 
@sebastienlagarde  what are your thoughts? We were just wanting to do some minimal effort here.

Reply:
I am just afraid this will be more jarring than the issue is actually trying to fix when using dynamic resolution. 

The pop will be very strong as we suddenly jump between very different resolutions and potentially in a flickering manner (as we hover the threshold).